store/src/db/sql.rs

```diff
@@ -275,6 +276,9 @@ pub fn select_notes(conn: &mut Connection) -> Result<Vec<Note>> {
         let merkle_path_data = row.get_ref(6)?.as_blob()?;
         let merkle_path = deserialize(merkle_path_data)?;
 
+        let details_data = row.get_ref(7)?.as_blob()?;
+        let details = deserialize(details_data)?;
```

Comment:
Related to the above comment: if we make column nullable and store `NULL` on `None`, we might need to handle this slightly differently.

Reply:
Does that only apply to notes or does it also apply to accounts? I have the understanding that we are not following this pattern when inserting accounts: https://github.com/0xPolygonMiden/miden-node/blob/f5a7bd9611ce56a4511030171389916a2ca1311b/store/src/db/sql.rs#L159-L214

Reply:
@phklive if I understood you correctly, in line #205 I process `None` details as `NULL`.

Reply:
> @phklive if I understood you correctly, in line 205 I process `None` details as `NULL`.

using `as_ref` ?